backend/envconfig/envconfig.go

```diff
@@ -72,6 +74,14 @@ func setFieldValue(field reflect.StructField, fieldValue reflect.Value, newValue
 		}
 		fieldValue.SetBool(boolean)
 		return nil
+	case reflect.Int64:
+		// right now the only env var of type int64 is AuthTokenLifetime
```

Comment:
there is no reflect.Kind of type time.Duration

Reply:
<img width="496" alt="Screen Shot 2020-04-10 at 9 35 17 AM" src="https://user-images.githubusercontent.com/3537801/79006743-af5ba980-7b0e-11ea-96d9-5fb7aff2466a.png">

Reply:
I guess we will have a function called process `int`. Inside which, we will do some `if else`.

Reply:
Please check the value of `Type.PkgPath` and `Type.Name`